djangify/djangify.py

```diff
@@ -154,8 +162,16 @@ def main():
     if directory is None:
         directory = os.getcwd()
 
+    # If encoding is not passed in as an argument or passed but is invalid one 
+    # use the locale encoding used for text data, according to user preferences.
+    # [Python Docs](https://docs.python.org/3/library/locale.html#locale.getpreferredencoding)
+    if (encoding is None or encodings.search_function(encoding) is None):
+        encoding = locale.getpreferredencoding()
+        
+
     logging.info("Directory : " + str(directory))
-    logging.info("app_name  : " + str(app_name))
+    logging.info("app_name  : " + str(APP_NAME))
+    logging.info("Encoding  : " + str(encoding))
```

Comment:
Let us not log encoding, we do not need the user to know about this, although `log.debug` could be used here.

Reply:
i'm changing this but I don't understand what's different for "Encoding". Why user is informed about "app_name" and "directory" again, even though he set this value himself. Why that isnt applies for "Encoing"  too.